src/Microsoft.AspNetCore.Sockets/Internal/TransferFormatFeature.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Sockets.Features;
+
+namespace Microsoft.AspNetCore.Sockets.Internal
+{
+    public class TransferFormatFeature : ITransferFormatFeature
```

Comment:
Nuke this DefaultConnection already implements it.

Reply:
Yep, just missed it.